doc/developer-notes.md

```diff
@@ -282,6 +283,20 @@ configure option adds `-DDEBUG_LOCKORDER` to the compiler flags. This inserts
 run-time checks to keep track of which locks are held and adds warnings to the
 `debug.log` file if inconsistencies are detected.
 
+### DEBUG_ADDRMAN
+
+Defining `DEBUG_ADDRMAN` performs frequent (and expensive) consistency checks on
+the entire addrman data structure. To enable it, add this line in `src/addrman.h`
```

Comment:
You can also just add `-DDEBUG_ADDRMAN` to your CPPFLAGS, rather than modifying the source.

Reply:
Thanks, added.